libr/anal/esil.c

```diff
@@ -10,6 +10,8 @@
 #include <float.h>
 #include <fenv.h>
 
+#define R_NUM_NAN -1 /* XXX */
```

Comment:
There is `NAN` macro in `math.h` for MSVC

Reply:
```c
int main() {
	printf("isnan(NAN)=%d\n", isnan(NAN));
	printf("isnan(-1)=%d\n", isnan((double)-1));
	return 0;
}
```
gives
```
isnan(NAN)=1
isnan(-1)=0
```

Reply:
I know. Its a bad patch. Just to check if the windows build was failing for something else. Theres a pr fixing that nan thing. But we can try with this NAN on windows too